modules/notification/mail/mail.go

```diff
@@ -98,3 +99,14 @@ func (m *mailNotifier) NotifyIssueChangeAssignee(doer *models.User, issue *model
 		mailer.SendIssueAssignedMail(issue, doer, ct, comment, []string{assignee.Email})
 	}
 }
+
+func (m *mailNotifier) NotifyMergePullRequest(pr *models.PullRequest, doer *models.User, baseRepo *git.Repository) {
+	if err := pr.LoadIssue(); err != nil {
+		log.Error("pr.LoadIssue: %v", err)
+		return
+	}
+
+	if err := mailer.MailParticipants(pr.Issue, doer, models.ActionClosePullRequest); err != nil {
```

Comment:
I'm just realizing that `mailer.MailParticipants()` will also mail the users mentioned in the content of the PR. Is that OK or should I change it so it will only do so when action is create issue?